### PR TITLE
175 order detail ajilgaand orullah

### PR DESCRIPTION
**Pull request by 14Berhee**
Screenshot 2024-12-05 at 21 57 12
Screenshot 2024-12-05 at 21 57 23

### CODE REVIEW

File: src/components/AddOrderModal.tsx [hunk 1]
```diff
@@ -7,23 +7,38 @@ import {
   DialogTrigger,
 } from "@/components/ui/dialog";
 import { SelectQuantity } from "@/components/basket-drawer/SelectQuantity";
+import PlusSign from "./ui/PlusSignSvg";
+import { Food } from "@/Providers/CartProvider";
+
+type menuItemProps = {
+  menuItem?: Food; // Made menuItem optional for safety
```
Comment (Baatarvant) [inline src/components/AddOrderModal.tsx:14]: Ene food geed tupe-iin MenuItem bolgoodhooch

File: src/components/restaurant/MenuItemLastCard.tsx [hunk 2]
```diff
@@ -1,54 +1,60 @@
...
       <div>
         <p>{menuItem.name}</p>
-        <div className="flex items-center gap-1 text-[14px] font-normal ">
+        <div className="flex items-center gap-1 text-[14px] font-normal">
           <p>{menuItem.price}$</p>
           <div className="w-[3px] h-[3px] bg-[black] rounded-full"></div>
           <LikeSvg />
-          <p>{91}%</p>
-          <p>{`(${1064})`}</p>
+          <p>91%</p>
```
Comment (Baatarvant) [inline src/components/restaurant/MenuItemLastCard.tsx:56]: Ene yun hatuu huvi bile?

**Review by Baatarvant — APPROVED** [review]

**Review by Baatarvant — CHANGES_REQUESTED** [review]
Comment bichiv

**Review by sumiyab — APPROVED** [review]

File: src/components/AddOrderModal.tsx [hunk 3]
```diff
@@ -7,23 +7,38 @@ import {
   DialogTrigger,
 } from "@/components/ui/dialog";
 import { SelectQuantity } from "@/components/basket-drawer/SelectQuantity";
+import PlusSign from "./ui/PlusSignSvg";
+import { Food } from "@/Providers/CartProvider";
+
+type menuItem = {
```
Comment (sumiyab) [inline src/components/AddOrderModal.tsx:13]: ene type neriig Tomoor bichiii

**Review by sumiyab — CHANGES_REQUESTED** [review]

**Review by Baatarvant — APPROVED** [review]

**Pull request merged**